R/start_transcription.R

```diff
@@ -27,14 +50,38 @@ function(
     format = tools::file_ext(url),
     language = "en-US",
     hertz = NULL,
+    output_bucket = NULL,
+    channel_identification = NULL,
+    show_speaker_labels = NULL,
+    max_speaker_labels = NULL,
+    vocabulary_name = NULL,
     ...
```

Comment:
Since there are more settings available, we could provide an optional argument `more_settings` taking a list which would be merged with `bod$Settings <- c(settings, more_settings)`. This will allow users to pass extra settings as needed. Theoretically, this is what the `...` is for but we already pass those to the HTTP request and I don't want to break compatibility.

This is optional though. If you don't have time to do it, I'll implement it after merging your PR.